src/main.js

```diff
@@ -255,7 +311,24 @@ class MiniMediaPlayer extends LitElement {
       ? `url(${this.config.background})`
       : this.thumbnail;
 
-    return html`<div class='cover' style='background-image: ${url};'></div>`;
+    if (this.config.artwork === 'material')
+      return html`<div class='cover' style='background-image: ${url}; background-color: ${this.backgroundColor}; width: ${this.cardHeight}px;'></div>`;
+    else
+      return html`<div class='cover' style='background-image: ${url};'></div>`;
```

Comment:
A function / getter (or a var over the if-else case) returning a styleMap with the computed styles for the `.cover` element would imo be cleaner, we could conditionally generate the styles over there and skip the if else case here.